include/swift/Basic/LangOptions.h

```diff
@@ -147,6 +147,9 @@ namespace swift {
     /// Disable API availability checking.
     bool DisableAvailabilityChecking = false;
 
+    /// Only check the availability of the API, ignore function bodies.
+    bool CheckAPIAvailabilityOnly = false;
```

Comment:
Can we merge this with `DisableAvailabilityChecking` to be an enum case of `AvailabilityCheckingScope`?

Reply:
ok. Thanks for the explanation!